start_cerberus.py

```diff
@@ -126,24 +141,40 @@ def main(cfg):
                 logging.info("Failed pods and components")
                 for namespace, failures in failed_pods_components.items():
                     logging.info("%s: %s", namespace, failures)
+                if slack_integration:
+                    failed_namespaces = ", ".join(list(failed_pods_components.keys()))
+                    slackcli.post_message_in_slack("In iteration %d, Cerberus found issues in "
+                                                   "namespaces: %s. Hence, setting the go/no-go "
+                                                   "signal to false." % (iteration,
+                                                                         failed_namespaces))
+                    slackcli.post_message_in_slack("Have a look at the Cerberus report below!")
+                    slackcli.post_file_in_slack()
+            else:
+                if slack_integration:
+                    slackcli.post_message_in_slack("No failures encounted in iteration: %d :tada:"
```

Comment:
Better to send messages only on failures to avoid spamming the channel with bunch of messages when we run the tool for longer durations. Thoughts?

Reply:
Yeah, that makes sense.